hacktoberfest/connector-contributor-guide.md

```diff
@@ -0,0 +1,68 @@
+# Connector Contributor Guide - Ballerina Hacktoberfest 2024
```

Comment:
Move the H1 heading to the frontmatter section.

Reply:
Add the frontmatter data:-

---
title: Verify Ballerina artifacts
description: The sections below include information about verifying Ballerina artifacts.
keywords: ballerina, verifying ballerina
intro: The sections below include information about verifying Ballerina artifacts. (optional)
---

Reply:
Added in https://github.com/ballerina-platform/ballerina-dev-website/pull/9502/commits/6821584a059c8c62bdf397f46eb814da278d9549